CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,169 @@
+# Contributor Ladder
+
+- [Contributor Ladder](#contributor-ladder-template)
+    - [Community Participant](#community-participant)
+    - [Contributor](#contributor)
+    - [Organization Member](#organization-member)
+    - [Reviewer](#reviewer)
+    - [Maintainer](#maintainer)
+- [Inactivity](#inactivity)
+- [Involuntary Removal](#involuntary-removal-or-demotion)
+- [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+- [Contact](#contact)
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Privileges" are things contributors on that level are entitled to.
+
+### Contributor
+
+_Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. Non-code contributions are equally, if not more, valued. People at the Contributor level may be new contributors, or they may only contribute occasionally._
+
+- Responsibilities:
+    - Follow the CNCF CoC
```

Comment:
https://github.com/cncf/foundation/blob/main/code-of-conduct.md

Reply:
Merging now, please open a PR to add the link.